script.js

```diff
@@ -0,0 +1,90 @@
+// define books as a collection of arrays
+let books = [
+  {
+    id: 1,
+    title: 'Lorem',
+    author: 'Testeroo Testyy',
+  },
+  {
+    id: 2,
+    title: 'Ipsum',
+    author: 'Testeroo Testyy',
+  },
+  {
+    id: 3,
+    title: 'Dolor',
+    author: 'Testeroo Testyy',
+  },
+];
+
+// Get DOM elements
+const bookListElement = document.getElementById('book-list');
+const addButtonElement = document.getElementById('add-btn');
+const titleInputElement = document.getElementById('title-input');
+const authorInputElement = document.getElementById('author-input');
+
+function addBook(title, author) {
+  const book = {
+    id: books.length + 1,
+    title,
+    author,
+  };
+  books.push(book);
+  localStorage.setItem('books', JSON.stringify(books));
+}
+
+function removeBook(id) {
+  books = books.filter((book) => book.id !== id);
+  let localStorageBooks = JSON.parse(localStorage.getItem('books'));
+  localStorageBooks = localStorageBooks.filter((obj) => obj.id !== parseInt(id, 10));
+  localStorage.setItem('books', JSON.stringify(localStorageBooks));
+}
+
```

Comment:
- [ ] Excellent work for the removeBook function. However, I can't delete the first book in local storage 👇🏻 and it causes when I reload the page it comes back, please let's fix that. 💯 👍🏻

![image](https://user-images.githubusercontent.com/79058364/235804367-cf937483-2a81-4eff-a304-eb57fa5ff505.png)

Reply:
hi 😎 @BenMukebo , thanks for your review but I don't have the same issue you brought up. Could you elaborate more on this? The remove button removes all books from the local storage whenever it's clicked without exceptions and even after reloading, the changes remain.

Reply:
Don't worry @Braingenius2, if so, the next code reviewer will check it out and approve your PR if everything is okay. 😉 👍🏻